hypothesis-python/tests/cover/test_slippage.py

```diff
@@ -59,6 +62,8 @@ def test(i):
     assert "ValueError" in output
 
 
+# no multiple failures on backend (yet?)
+@xfail_on_crosshair(Why.other)
 def test_shows_target_scores_with_multiple_failures():
```

Comment:
huh, yeah, iirc we flip back to the hypothesis backend _immediately_ on finding a failure, and then proceed immediately to shrinking.  Which I think is a totally reasonable design, but means that this test can be skipif-backend-not-hypothesis rather than a crosshair-specific xfail 🙂